src/common/styles/variables.scss

```
@@ -1,4 +1,4 @@
-$color_slategray: slategray;
-$color_white: white;
-$color_blue: blue;
-$color_red: red;
+$gray-6: #8c8c8c;
+$white: white;
```

```suggestion
#fff
```
Цвета всегда должны указываться, через HEX, а не названием
ТОЖЕ для blue, red!